src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/SymbolFactory.vb

```diff
@@ -56,6 +56,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             Return False
         End Function
 
+        Friend Overrides Function IsAcceptedOutAttributeModifierType(type As TypeSymbol) As Boolean
+            ' VB doesn't deal with ref-readonly parameters, function pointers, or return-types.
```

Comment:
>or return-types. [](start = 79, length = 16)

Is this accurate?

Reply:
I just copied the comment from the method above and added function pointers. I'll modify both to specify ref-return types, which is what I suspect it should have been in the first place.

---
In reply to: [409785521](https://github.com/dotnet/roslyn/pull/43355#discussion_r409785521) [](ancestors = 409785521)